adyenv6b2ccheckoutaddon/acceleratoraddon/web/src/com/adyen/v6/controllers/pages/AdyenSummaryCheckoutStepController.java

```diff
@@ -163,7 +163,7 @@ public String placeOrder(@ModelAttribute("placeOrderForm") final PlaceOrderForm
         //Validate the cart
         if (validateCart(redirectModel)) {
             // Invalid cart. Bounce back to the cart page.
-            return REDIRECT_PREFIX + "/cart";
+            return REDIRECT_PREFIX + AdyenControllerConstants.CART_PREFIX;
```

Comment:
We can import AdyenControllerConstants.CART_PREFIX and use CART_PREFIX everywhere.

Reply:
Done